src/phoenix/server/api/routers/v1/evaluations.py

```diff
@@ -158,3 +176,31 @@ async def _add_evaluations(
     for evaluation in encode_evaluations(evaluations):
         state.queue_evaluation_for_bulk_insert(evaluation)
         traces.put(evaluation, project_name=project_name)
+
+
+def _read_sql_span_evaluations_into_dataframe(
+    connectable: Connectable,
+    project_name: str,
+) -> DataFrame:
+    """
+    This function inputs a synchronous connection to pandas.read_sql since
+    it does not support async connections.
+
+    For more information, see:
+
+    https://stackoverflow.com/questions/70848256/how-can-i-use-pandas-read-sql-on-an-async-connection
+    """
+    return pd.read_sql(
+        select(models.SpanAnnotation, models.Span.span_id)
+        .join(models.Span)
+        .join(models.Trace)
+        .join(models.Project)
+        .where(
+            and_(
+                models.Project.name == project_name,
+                models.SpanAnnotation.annotator_kind == "LLM",
+            )
+        ),
+        connectable,
```

Comment:
just to help pandas out a little bit
```suggestion
        )
        .order_by(models.SpanAnnotation.name),
        connectable,
```

Reply:
turns out this is unnecessary